### ISSUE TITLE
Dysplastic Megalencephaly (DMEG)

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#7609]
**Preferred term label:**
Dysplastic megalencephaly 

**Synonyms**
DMEG
bilateral hemimegalencephaly

**Definition (free text, please give PubMed ID)**
A rare cerebral malformation characterized by bilateral cerebral hemisphere overgrowth and extensive malformation of cortical development.
PMID:29594439

**Parent term (use hpo.jax.org/app)**
Abnormal cerebral morphology HP:0002060

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MONDO:0035162 'PIK3CA-related overgrowth syndrome'

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-1493-865X (gully burns is the original requester)

related to https://github.com/monarch-initiative/mondo/issues/5210

**Comment by pnrobinson** [comment]
added as HP:0034329
